CVLByExample/Curve/certora/conf/runViewReentrancyBuiltinRule.conf

```diff
@@ -0,0 +1,20 @@
+{
+  "files": [
+    "contracts/Curve.sol",
+    "contracts/CurveTokenExample.sol",
+    "contracts/ERC20.sol"
+  ],
+  "verify": "Curve:certora/specs/BuiltinViewReentrancy.spec",
+  "link": [
+    "Curve:lp_token=CurveTokenExample",
+    "Curve:coins_1=ERC20"
+  ],
+  "msg": "Curve builtin linking coins_1 w/ view guard",
```

Comment:
msg should be more clear

Reply:
@DanieLion55 what did you /Omer want to say here?

Reply:
Changed according to what I understand.